### DISCUSSION TITLE
国际化umd无法打包问题

### DISCUSSION THREAD

**Discussion by neil-lei0921** [umijs/dumi#2026] [General]
按照官网初始化项目 -> src下添加locales文件夹以及zh-CN.json和en-US.json
![image](https://github.com/umijs/dumi/assets/92282065/28f0634f-3a2c-4e04-b6be-b473d99c0933)
配置.dumirc.ts
![image](https://github.com/umijs/dumi/assets/92282065/37be471e-3395-4cf6-a45b-2e03fd03c581)
配置.fatherrc.ts
![image](https://github.com/umijs/dumi/assets/92282065/597894e6-ffca-4a8c-9c42-e1ceb218dd51)
运行"build": "father build"无法打包
![image](https://github.com/umijs/dumi/assets/92282065/9f502552-2728-4d9d-8e32-d9a7cac7054d)
请问这个问题怎么解决？

**Comment by neil-lei0921** [comment]
并且在Foo里使用国际化 无法成功 页面会显示defaultMessage 没有defaultMessage的情况会显示为id
![image](https://github.com/umijs/dumi/assets/92282065/6789d015-18be-4f83-a438-b7b75f8753f0)
![image](https://github.com/umijs/dumi/assets/92282065/b0a869ef-3eb4-48b5-b038-b94cdaaee8aa)

**Comment by neil-lei0921** [comment]
@PeachScript 麻烦帮忙看看呢

**Comment by PeachScript** [comment]
dumi 提供的 `react-intl` 相关 API 仅仅是给组件库文档用的，以及任何 dumi 导出的 API 都不应在组件库源码中引用

组件自己的国际化要单独依赖 react-intl 相关库实现，或者参考 ant-design 的方式做国际化
